TKGame/Level Editor Content/Trigger.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Content;
+using Microsoft.Xna.Framework.Graphics;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TKGame.Level_Editor_Content
+{
+    internal class Trigger
+    {
+        #region Member Variables
+        private static readonly int screenWidth = 1600;
```

Comment:
Will this always be the pixel size of screens? Or do we need some way to get the pixel size of the current machine?